tests/ui/unit/init.js

```diff
@@ -1,18 +1,12 @@
 // Karma/webpack entry for tests
 
-import jQuery from 'jquery';
 // Manually import angular since angular-mocks doesn't do so itself
 import 'angular';
 import 'angular-mocks';
 import 'jasmine-jquery';
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
-// Global variables are set here instead of with webpack.ProvidePlugin
-// because neutrino removes plugin definitions for karma runs:
-// https://github.com/mozilla-neutrino/neutrino-dev/issues/617
-window.jQuery = jQuery;
```

Comment:
This manual `window.jQuery` assignment is no longer required since the karma run now inherits the same webpack plugins list as used during start/build, and therefore can use ProvidePlugin like they do.